composer.json

```diff
@@ -17,28 +16,53 @@
   ],
   "scripts": {
     "dev": "php -S 0.0.0.0:8000",
-    "analyse": "vendor/bin/phpstan analyse --ansi",
+    "phpstan": "vendor/bin/phpstan analyse --ansi",
     "rector": "vendor/bin/rector process --dry-run --ansi",
     "rector:fix": "vendor/bin/rector process --ansi",
     "style": "vendor/bin/ecs check --ansi",
-    "style:fix": "vendor/bin/ecs check --fix --ansi"
+    "style:fix": "vendor/bin/ecs check --fix --ansi",
+    "pest": "./vendor/bin/pest --stop-on-defect --colors=always",
+    "pest:parallel": "./vendor/bin/pest --colors=always --parallel",
+    "pest:profiling": "./vendor/bin/pest --colors=always --profile",
+    "pest:coverage": "./vendor/bin/pest --colors=always --coverage",
+    "pest:type-coverage": "./vendor/bin/pest --colors=always --type-coverage --min=99",
+    "test": [
+      "@test:style",
+      "@test:rector",
+      "@phpstan",
+      "@pest:type-coverage",
+      "@pest:parallel"
+    ],
+    "core:update": [
+      "composer update --no-interaction --prefer-stable",
+      "composer bump --ansi",
+      "composer validate --strict --ansi",
+      "composer audit --ansi"
+    ]
   },
   "require": {
-    "php": ">=8.1",
+    "php": ">=8.2",
```

Comment:
This means that users using php 8.1 won't be able to use php-icons, no?
I think we should leave it to 8.1 as it is still supported till 31 Dec 2025.

I'm guessing you added this because of pest v3?
I guess we could have a dev environment with php 8.2 for pest and release a php8.1 compatible version of php-icons.

Reply:
Yes you're right. I'm gonna rollback Pest to `^2.0` and switch to `^3.0` later in 2026 :)